src/distortion/barrel-distortion-fragment-v2.js

```diff
@@ -42,7 +42,7 @@ var BarrelDistortionFragment = {
     'varying vec2 vUV;',
 
     'float poly(float val) {',
-      'return (showCenter == 1 && val < 0.00005) ? ',
+      'return (showCenter == 1 && val < 0.00010) ? ',
```

Comment:
Yes. On many lower res devices, the dots were not big enough to be even 1 pixel wide so they were not showing.